bin/rudder-client

```diff
@@ -14,7 +14,7 @@ fi
 CURL_OPTS="--tlsv1.3 --location --fail --connect-timeout 10"
 ERROR_OPT="--show-error"
 CURL_VERBOSITY="--silent"
-EXEC="exec"
+EXEC=""
```

Comment:
why?

Reply:
we can probably remove the variable

Reply:
it is used in verbose mode to dump the curl command